packages/python/plotly/plotly/data/__init__.py

```diff
@@ -1,32 +1,72 @@
 """
 Built-in datasets for demonstration, educational and test purposes.
 """
+import os
+from importlib import import_module
 
+import narwhals.stable.v1 as nw
 
-def gapminder(datetimes=False, centroids=False, year=None, pretty_names=False):
+AVAILABLE_BACKENDS = {"pandas", "polars", "pyarrow"}
```

Comment:
If that's of interest, adding modin and cudf is also quite straightforward and all the operations are supported via narwhals